ccd-definition/FixedLists/CareSupervision/manage-documents/FurtherEvidenceTypeLA.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "FurtherEvidenceTypeLA",
```

Comment:
might update name, as this might be confusing in next story (https://tools.hmcts.net/jira/browse/FPLA-2768)